conf/gear.conf

```diff
@@ -63,7 +63,11 @@ gearpump {
   ### If you don't know what this is about, don't change it
   ###########################
 
-  task-dispatcher {
+  task-dispatcher = "shared-thread-pool"
```

Comment:
The value should be full config key, and better use -dispatcher as suffix.

For example, For Akka-remoting, it use the dispatcher like this:
```
akka.remote.use-dispatcher = "akka.remote.default-remote-dispatcher"
```